packages/claims/src/claims/claims.ts

```diff
@@ -63,32 +67,71 @@ export class Claims implements IClaims {
   }
 
   /**
-   * Verifies integrity of the claim, the claim is issued by the user
-   *  delegate and the authenticity of the issuer's signature
+   * Verifies issuance and publishing of claim at `claimUrl`
    *
    * @param claimUrl {string}
    * @param hashFns {{ [alg: string]: (data: string) => string }}
    */
   async verify(
-    claimUrl: string, hashFns?: { [alg: string]: (data: string) => string },
+    claimUrl: string,
+    {
+      hashFns,
+      issuerDoc,
+      holderDoc,
+    }: {
+      hashFns?: { [alg: string]: (data: string) => string };
+      issuerDoc?: IDIDDocument;
+      holderDoc?: IDIDDocument;
```

Comment:
This assumes a rename of `PubKeyType` to `PubKeyPurpose`
```suggestion
      hashFns,
      issuerDoc,
      holderDoc,
      verificationPurpose,
    }: {
      hashFns?: { [alg: string]: (data: string) => string };
      issuerDoc?: IDIDDocument;
      holderDoc?: IDIDDocument;
      verificationPurpose?: pubKeyRelationship
```